client-py/market_maker_bot.py

```diff
@@ -122,18 +124,11 @@ def clamp(value: Decimal):
                 )
             )
             await client.send(create_order)
-
-    await iteration()
+    
     while True:
-        _kind, message = await client.recv()
-        if isinstance(message, RequestFailed):
-            logger.error(
-                f"{message.request_details.kind} request failed: {message.error_details.message}"
-            )
```

Comment:
Can we leave this in, just using the messages returned by `get_buffered_messages`? otherwise looks good